app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -209,9 +209,23 @@ var update_registry = function() {
   var denomination = $('#token_name').text();
   var original_amount = $('#original_amount').val();
   var net = round(original_amount - tc, 2);
+  var over = round((original_amount - get_total_cost()) * -1, 4);
+  var addr = web3.eth.coinbase.substring(38);
 
   $('#total_cost').html(tc + ' ' + denomination);
   $('#total_net').html(net + ' ' + denomination);
+  $('#total_overage').html(over + ' ' + denomination);
```

Comment:
This section I feel is unnecessary in case `over < 0` right? Since it will anyways be hidden. Can't we put this inside the condition check to optimize the code?

Reply:
That would be correct and it makes absolute sense to do so. I've restructured it in the new commit!